mantis-common/src/main/java/com/mantisrx/common/utils/Services.java

```diff
@@ -21,20 +21,23 @@
 import io.mantisrx.shaded.com.google.common.util.concurrent.Service.State;
 import java.util.concurrent.CompletableFuture;
 import java.util.concurrent.Executor;
+import lombok.extern.slf4j.Slf4j;
 
+@Slf4j
 public class Services {
 
     /**
-     * Equivalent of service.startAsync().awaitRunning() except that this provides a future that completes when
-     * the service reaches the RUNNING state. if the service on the other hand fails to start, then the future is completed
-     * exceptionally with the Throwable cause.
+     * Equivalent of service.startAsync().awaitRunning() except that this provides a future that
+     * completes when the service reaches the RUNNING state. if the service on the other hand fails
+     * to start, then the future is completed exceptionally with the Throwable cause.
      *
-     * @param service service that needs to be started
+     * @param service  service that needs to be started
      * @param executor executor to be used for notifying the caller.
      * @return a future
      */
     public static CompletableFuture<Void> startAsync(Service service, Executor executor) {
-        Preconditions.checkArgument(service.state() == State.NEW, "Assumes the service has not been started yet");
+        Preconditions.checkArgument(service.state() == State.NEW,
```

Comment:
is this call thread-safe? is lock needed?

Reply:
This is the existing code. Not related to my changes.